Configuration.props

```diff
@@ -135,7 +135,7 @@
     <AndroidToolPath Condition=" '$(AndroidToolPath)' == '' ">$(AndroidSdkFullPath)\tools</AndroidToolPath>
     <AndroidToolsBinPath Condition=" '$(AndroidToolsBinPath)' == '' ">$(AndroidToolPath)\bin</AndroidToolsBinPath>
     <AndroidToolExe Condition=" '$(AndroidToolExe)' == '' ">android</AndroidToolExe>
-    <EmulatorVersion Condition=" '$(EmulatorVersion)' == '' ">5264690</EmulatorVersion>
+    <EmulatorVersion Condition=" '$(EmulatorVersion)' == '' ">5395263</EmulatorVersion>
     <EmulatorToolPath Condition=" '$(EmulatorToolPath)' == '' ">$(AndroidSdkFullPath)\emulator</EmulatorToolPath>
     <EmulatorToolExe Condition=" '$(EmulatorToolExe)' == '' ">emulator</EmulatorToolExe>
```

Comment:
If we want to try the headless emulator, shouldn't we *also* modify this line to use `emulator-headless`?

Reply:
I didn't want to do it in this PR - it is to bring the new version in, then we should test it locally and only then make the changes here.

Reply:
And it seems we need to get #3000 in first, to get real test results here...

Reply:
Looks like the `5395263` version of the emulator-darwin-5395263.zip does NOT contain `emulator-headless` :/